app/helpers/edit_field_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: false
+module EditFieldHelper
+  def reorder_languages(languages)
+    front_languages = %w[English Portuguese Spanish]
```

Comment:
I expect we'll have a similar request from other ephemera projects who want other languages at the top of the list.  Can we move the list of languages to the ephemera project?

Reply:
You want a new field on the project and a UI for setting these?

Reply:
I'm open to other ways of configuring this if there's something easier, but I definitely expect that different ephemera projects will want different languages to be at the top of the list.

Reply:
Okay. Letting them put in what they want is probably better than making them ask us. The whole thing is strange to me since I am keyboard-oriented and would just start typing :)